core/src/test/java/com/kin/ecosystem/core/data/order/OrderRepositoryTest.java

```diff
@@ -548,6 +550,7 @@ public void onFailure(KinEcosystemException exception) {
 
 		verify(blockchainSource).removePaymentObserver(observersList.get(0));
 		verify(blockchainSource).removePaymentObserver(observersList.get(1));
+		verify(remote).changeOrder(anyString(), any(Body.class), any(Callback.class));
```

Comment:
can't you verify the actual input? making sure they match the expected ones?

Reply:
nope, the `Body` and `Callback` created inside the repo.